### ISSUE TITLE
Horizontal scroll bar in right hand post/block settings bar when using firefox.

### ISSUE THREAD

**Issue by oldrup** [WordPress/gutenberg#29840]
## Description
The Gutenberg Post/Block settings bar displays a horizontal scroll bar in firefox, once enough settings are opened to cause er vertical scroll bar. The horizontal scroll bar does not appear in chrome.

## Step-by-step reproduction instructions

1. edit a post using firefox for windows
2. toggle open enough options in the right hand settings bar to activate the vertical scroll bar

## Expected behaviour
Vertical scroll bar appears
Settings bar is resized to facilitate the scroll bar?

## Actual behaviour
Vertical scroll bar appears
Horizontal scrollbar also appears

## Screenshots or screen recording (optional)
![image](https://user-images.githubusercontent.com/17141205/110993208-48c2b980-8377-11eb-8d62-67318c87b66e.png)

## WordPress information
WordPress 5.7

## Device information
Firefox (lates stable) on Windows 10 (fully updated9

**Comment by cngodles** [comment]
I can verify this is happening with the latest Firefox and the latest WordPress.

If I remove the following style using browsers tools, the horizontal scroll bar goes away:
`.interface-interface-skeleton__sidebar {
    overflow: auto;
}`

![image](https://user-images.githubusercontent.com/5685764/110998250-8a8a3a80-834c-11eb-864f-26fb604daac0.png)

**Comment by t-hamano** [comment]
@oldrup

Sorry for the very late reply.
I have tested this issue again and could not reproduce it.
Can anyone test this issue?

## My environment

- OS: Windows 11
- Browser: Firefox 102.0.1 (64bit)
- WordPress Version: 6.0.1
- Gutenberg Version: 13.7.0-rc.2 (with or without)

**Comment by oldrup** [comment]
> Can anyone test this issue?

Sure. Still got it. No changes there since originally posted. [22 sec screencast:](https://www.loom.com/share/77c47277d2d847e1a24d11775049052b) 

- OS: Windows 10
- Browser: Firefox 102
- WordPress Version 6.0.1
- Gutenberg: Shipped with WP 6.0.1

Was able to [choose a narrower scroll bar](https://www.makeuseof.com/change-firefox-scrollbar-style/) using Firefox:config flags, and it sort of mitigated the issue, but it still an eyesore.

So weird you aren't able to reproduce :D Have this on every single Firefox install and site I've edited since, forever :D

- No problems in Chromium based browsers.
- Unable to test in Safari.

Please let me know if I in any way can assist in improving the Firefox / Gutenberg experience.

**Comment by t-hamano** [comment]
Thank you for your replay.
I haven't been able to reproduce it yet, but I will first mentions #31858 because I think this PR probably has something to do with it.

**Comment by t-hamano** [comment]
I tested this issue again with the latest Gutenberg and still could not reproduce it. Horizontal scrolling does not occur regardless of the thickness of the scrollbar.

https://user-images.githubusercontent.com/54422211/206827874-3d58d476-1565-4f49-8f7a-0283ce2611ed.mp4

> If I remove the following style using browser tools, the horizontal scroll bar goes away:
> .interface-interface-skeleton__sidebar { overflow: auto; }

As the video above attempts to show, removing this style will also scroll the content area itself.

@oldrup 
How about disabling all plugins or changing the theme and see if that solves the problem?

**Comment by oldrup** [comment]
Hey

Thank you for looking into this.

I've got a feeling it's a Windows only issue. Haven't got a Mac to verify that.

It's been an issue on the lots of different sites I've been working on, since I've created this ticket. Regardless of theme and the lack of plugins on new sites.

However. I typically add custom CSS to my sites at some point, to improve the user experience front and backend. And I found that this little snippet, solves the issue for me, by allocation space for vertical scrollbar at all times, and this not showing a horizontal..

`
*** Firefox vs Gutenberg sidebar scrollbar fix ***/
.interface-interface-skeleton__sidebar {
  scrollbar-gutter: stable;
}
`

For more info: https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter

That is good enough for me, and I'll close the issue. There  are bigger fish to fry.

Thanks again.
